core/modules/taxonomy/taxonomy.api.php

```diff
@@ -272,6 +272,23 @@ function hook_taxonomy_term_view_alter(&$build) {
   $build['#post_render'][] = 'my_module_taxonomy_term_post_render';
 }
 
+/**
+ * Alter the results of taxonomy_term_page()
+ *
+ * This hook is called after the node listing for a taxonomy term page has been assembled in a structured array
+ * and may be used for altering that listing.
```

Comment:
We should stick to the rule of 80 characters max per line:

```suggestion
 * This hook is called after the node listing for a taxonomy term page has
 * been assembled in a structured array and may be used for altering that
 * listing.
```

See https://docs.backdropcms.org/doc-standards#general

> Lines containing comments (including docblocks) must wrap as close to 80 characters as possible without going over, ...